### DISCUSSION TITLE
Network Switches

### DISCUSSION THREAD

**Discussion by SebUll** [AsBuiltReport/AsBuiltReport.GitHub.Discussions#3] [Q&A]
Hello everyone,

Is there a possibility to create and establish such a query for switches?
I would be very pleased to receive a positive answer.

Yours sincerely
Sebastian

**Comment by tpcarman** [comment]
If it's possible to extract information via a REST API or PowerShell then it is possible to create a report. 

What switches were you thinking of?

**Reply by SebUll** [reply]
Hi,

That sounds very good. It would be about Mellanox, Aruba, HPE, Dell and Cisco network switches. If you need more detailed information on the commands or REST API I can certainly send you some information.

**Reply by alagoutte** [reply]
Duplicate with https://github.com/orgs/AsBuiltReport/discussions/4

and it will be complicated to same report for multiple switch vendor...

**Reply by tpcarman** [reply]
For anyone who might be interested in creating a report module, please refer to the [documentation](https://www.asbuiltreport.com/dev-guide/creating-a-report-module/).
